client/src/components/Sidebar/index.tsx

```diff
@@ -1,29 +1,21 @@
 import React from "react";
+import classNames from "classnames";
 import { SidebarButton } from "@components/core";
 import useSidebar from "@components/Sidebar/useSidebar";
+import styles from "./styles.module.scss";
 
 const Sidebar = () => {
 	const { isHidden, toggleSidebar } = useSidebar();
 
+	const sidebarClass = classNames(styles.sidebar, {
+		[styles.isDisplayed]: !isHidden,
+	});
+
 	return (
-		<div
-			className="Sidebar"
-			style={
-				isHidden
-					? {}
-					: {
-							background: "ghostwhite",
-							width: "20%",
-							height: "100vh",
-							textAlign: "left",
-							display: "flex",
-							flexDirection: "column",
-							justifyContent: "space-around",
-						}
-			}
-		>
+		<div className={sidebarClass}>
 			{!isHidden && (
 				<div>
+					<span className={styles.boards}>ALL BOARDS</span>
```

Comment:
I'll add a todo for this to render the count of the boards per design: 
![1](https://github.com/Erikang20/Kanban-task-management-web-app/assets/13842841/34076ae9-c4a6-4fc4-aa57-2191044aabd6)